app/src/main/java/com/codesoom/assignment/http/HttpStatus.java

```diff
@@ -0,0 +1,19 @@
+package com.codesoom.assignment.http;
+
+public enum HttpStatus {
+    OK(200),
+    CREATED(201),
+    NOT_FOUND(404),
+    METHOD_NOT_ALLOWED(405),
+    INTERNAL_SERVER_ERROR(500);
+
+    private int status;
```

Comment:
`final`로 불변성을 강조해 주면 좋겠습니다.
```suggestion
    private final int status;
```